app/components/autocomplete/emoji_suggestion/emoji_suggestion.tsx

```diff
@@ -70,11 +70,12 @@ type Props = {
     maxListHeight: number;
     updateValue: (v: string) => void;
     onShowingChange: (c: boolean) => void;
-    rootId: string;
+    rootId?: string;
     value: string;
     nestedScrollEnabled: boolean;
     skinTone: string;
-    hasFilesAttached: boolean;
+    hasFilesAttached?: boolean;
+    inPost?: boolean;
```

Comment:
the component does not seem to be receiving this prop from anywhere, do we really need it?

Reply:
We need it to make sure we don't react to the latest post if we are not in the post draft. But it was not wired to the Autocomplete component correctly. Will fix.